internal/resources/grafana/resource_sso_settings.go

```diff
@@ -34,25 +35,30 @@ Manages Grafana SSO Settings for OAuth2. SAML support will be added soon.
 		ReadContext:   ReadSSOSettings,
 		UpdateContext: UpdateSSOSettings,
 		DeleteContext: DeleteSSOSettings,
-		Importer: &schema.ResourceImporter{
-			StateContext: schema.ImportStatePassthroughContext,
-		},
```

Comment:
The importer doesn't work with the current changes on Read(), so I'm removing it for now.